id/mod.go

```diff
@@ -10,6 +11,12 @@ type Id interface {
 	GetLength() int
 	GetBase() byte
 	GetDigit(pos int) byte
+	CommonPrefix(other Id) Prefix
+	PrefixUntilFirstDifferentDigit(other Id) (Prefix, error)
+}
+
+func BaseAndLenFromPlayers(numPlayers int) (base byte, len int) {
+	return 16, 3
```

Comment:
A dummy implementation. This method is where the id space computation happens, the first actual implementation will probably be one for hashed ids.

Reply:
Different approaches to this computation to be explored in a separate PR.